ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/InvitationBubble.qml

```diff
@@ -18,11 +18,22 @@ Item {
     width: rectangleBubbleLoader.width
 
     function getCommunity() {
-        let community = JSON.parse(chatsModel.communities.list.getCommunityByIdJson(communityId));
```

Comment:
Why are we using json here?

Reply:
TBH, I don't remember. I didn't code the original